test_pkgs.txt

```diff
@@ -17,7 +17,10 @@ github.com/TRON-US/go-btfs/core/commands/rm
 github.com/TRON-US/go-btfs/core/commands/storage
 github.com/TRON-US/go-btfs/core/commands/storage/challenge
 github.com/TRON-US/go-btfs/core/commands/storage/helper
-github.com/TRON-US/go-btfs/core/commands/storage/upload
+github.com/TRON-US/go-btfs/core/commands/storage/upload/escrow
+github.com/TRON-US/go-btfs/core/commands/storage/upload/helper
```

Comment:
the upload/escrow contains escrow related routines

but upload/guard does not exist, it's hard to find where guard calls are being made.. could you move them into upload/guard instead of upload/helper which is still too general...